### PR TITLE
Set up Shopify Theme and add Fonts

### PR DESCRIPTION
**Pull request by chrismekim**
### Description

- This PR removes most of the previous set up when it was going to be a Sanity React app, and sets up a blank Shopify theme. THANKS @sepowitz !!!
- Adds fonts and typography styling.
- Updates readme and Pull Request template
 
### Type(s) of changes

- [x] New feature

### How Has This Been Tested?

https://lpjkd1l3u8kiqal7-52674822324.shopifypreview.com

### CODE REVIEW

File: theme/styles/settings/_typography.scss [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+  font-size: 1rem;
+  line-height: 1.171875rem;
+  letter-spacing: 0.0275rem;
+}
+
+.sans-xxs {
+  font-size: 0.75rem;
+  line-height: 0.83125rem;
+  letter-spacing: 0.0275rem;
+}
```
Comment (chrismekim) [inline theme/styles/settings/_typography.scss:81]: Names of the styles match the ones in Figma. Thought it would be easier to keep track of them!

**Comment by chrismekim** [comment]
Question: ok to leave the templates as is and change as we develop?

File: theme/styles/settings/_typography.scss (hunk 1 above)
Comment (sepowitz) [reply theme/styles/settings/_typography.scss:81]: I'm into it!

**Review by sepowitz — COMMENTED** [review]
Yah, I think it's cool to leave templates as is for now

**Review by sepowitz — APPROVED** [review]

**Review by conordavidson — APPROVED** [review]

**Pull request merged**